### PR TITLE
Adds dialog after center creation. Fixes #470

### LINKED ISSUE #470: Feature Addition in Center Creation
Summary:  Similar to issue #308, on creating a new Center the page remains as it is without refreshing. A dialog should appear notifying the status and giving an option to create a new Center or go back to search fragment.

Expected Behaviour: A dialog gets displayed on successful creation of a Center which asks to create a new Center or not. On click of create new Center, the  previously entered data gets removed so new details can be entered. On clicking no, the search fragment is displayed again.

Observed Behaviour: The Center creation page remains as it is, even on successful creation of a Center

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java [hunk 1]
```diff
@@ -151,6 +157,25 @@ private void initiateCenterCreation(CenterPayload centerPayload) {
 
         if (isCenterNameValid()) {
             mCreateNewCenterPresenter.createCenter(centerPayload);
+            AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(getActivity());
+            alertDialogBuilder
+                    .setMessage("Would you like to create another center?")
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java]: Please don't use hardcoded strings.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java [hunk 2]
```diff
@@ -151,6 +157,25 @@ private void initiateCenterCreation(CenterPayload centerPayload) {
...
+                    .setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+                        @Override
+                        public void onClick(DialogInterface dialogInterface, int i) {
+                            et_centerName.setText("");
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java]: Please follow the coding style conventions. 
If the files you are working on don't follow the coding style conventions, considering changing them conforming to the rules stated at https://github.com/openMF/android-client/wiki/Code-style-conventions

**Comment by tarun0** [comment]
Also, every fix should be sent in a single commit. So you'll need to squash the commits into one after making all the changes from your end.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java (hunk 1 above)
Comment (shobhitagarwal1612) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java]: Ok, I'll make the changes

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java (hunk 2 above)
Comment (shobhitagarwal1612) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java]: Thanks

**Comment by shobhitagarwal1612** [comment]
@tarun0 Please review the changes

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java [hunk 3]
```diff
@@ -151,6 +153,29 @@ private void initiateCenterCreation(CenterPayload centerPayload) {
...
+                    R.style.MaterialAlertDialogStyle);
+            alertDialogBuilder
+                    .setMessage(getString(R.string.create_center_dialog))
+                    .setPositiveButton(getString(R.string.action_yes),
+                            new DialogInterface.OnClickListener() {
+                                @Override
+                                public void onClick(DialogInterface dialogInterface,
+                                                    int i) {
+                                    et_centerName.setText("");
+                                    cb_centerActiveStatus.setChecked(false);
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java]: Rename these to `etCenterName`, `cbCenterActiveStatus` and similarly, `btnSomeButton` `tvSomeTextView` etc for the other views in this file.

**Comment by therajanmaurya** [comment]
I had a talk with @edcable we do not need to create client group and center more than one. Need to open as client group or center created successfully created. 
Resolved in #520

**Pull request closed without merging**